src/components/audio-feedback.js

```diff
@@ -1,20 +1,13 @@
 AFRAME.registerComponent("networked-audio-analyser", {
   schema: {},
   async init() {
-    const networkedEl = await NAF.utils.getNetworkedEntity(this.el);
-    const ownerId = networkedEl.components.networked.data.owner;
-
-    const stream = await NAF.connection.adapter.getMediaStream(ownerId);
-
-    if (!stream) {
-      return;
-    }
-
-    const ctx = THREE.AudioContext.getContext();
-    const source = ctx.createMediaStreamSource(stream);
-    this.analyser = ctx.createAnalyser();
-    this.levels = new Uint8Array(this.analyser.frequencyBinCount);
-    source.connect(this.analyser);
+    this.el.addEventListener("sound-source-set", event => {
```

Comment:
I'd prefer a model wher eyou got the `networked-audio-source` component and asked it for a source, but an evented model works too. Probably just down to personal preference .

Reply:
I wanted to avoided reaching into `el.components`. Feels like bad practice.

Reply:
I don't think its really bad practice to have components depend on other components. Though I do thnk it would be not ideal to reach directly into that other components data. I assumed the stream might be exposed by a function that returned a promise to a stream.

Reply:
Hmm, yeah, fair point. I'll bring it up on the NAF PR.